### DISCUSSION TITLE
Appel à votre créativité pour le nouveau logo du Meetup Python de Grenoble

### DISCUSSION THREAD

**Discussion by Pierre-Loic** [meetup-python-grenoble/meetup-python-grenoble.github.io#71] [Announcements]
Nous faisons appel à la communauté pour choisir le **nouveau logo du Meetup Python de Grenoble**. Vous avez **jusqu'au 27 avril 2023** pour soumettre une **proposition de logo**, un vote sera effectué durant le [Meetup Python du 27 avril 2023](https://meetup-python-grenoble.github.io/events/2023-04-27/).
N'étant pas tous des graphistes dans l'âme, nous vous proposons d'utiliser des **outils d'intelligence artificielle générative** pour créer le logo (par exemple [DALL-E](https://openai.com/product/dall-e-2), [Midjourney](https://docs.midjourney.com/docs/quick-start) ou [Stable Diffusion](https://stablediffusionweb.com/)).
Les **critères** qui doivent être représentés sur le logo sont :
- les différents **domaines du Python** : web, data, intelligence artificielle...
- la localisation à **Grenoble** ou aux alentours (patrimoine, géographie, histoire...)
- au moins **une des valeurs** retranscrites dans le [zen du Meetup Python de Grenoble](https://meetup-python-grenoble.github.io/zen/)

Vous pouvez **soumettre vos propositions de logos** directement dans cette **discussion Github**. 

**A vos prompts !**

**Comment by rclement** [comment]
Voici un logo généré avec DALL-E à partir du logo Python officiel et du prompt suivant : _An oil painting of mountains behind the blue and yellow snakes_

![DALL·E 2023-03-30 11 22 27 - An oil painting of mountains behind the blue and yellow snakes](https://user-images.githubusercontent.com/1238873/228791453-37097a90-0fcc-42d5-8b53-b6eb1aa5b82f.png)

**Comment by jbbarre** [comment]
L'image d'épinal de Grenoble en version pytonique.
![grePython](https://user-images.githubusercontent.com/54904520/228850442-7c3e4b12-ca75-4549-a692-0c6e4e902cb3.png)

**Comment by Pierre-Loic** [comment]
En demandant à DALLE d'ajouter des montagnes, ça donne ça :
![logo_v4](https://user-images.githubusercontent.com/37104025/232317497-53b5b8a9-1868-4184-86e5-b9ed6c5f1af9.png)
